registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-confirm.component.ts

```diff
@@ -17,10 +19,14 @@ import * as globalConst from '../../core/services/globalConstants';
   schemas: [CUSTOM_ELEMENTS_SCHEMA]
```

Comment:
is this needed?

Reply:
Yes, to allow angular components in html